.github/ISSUE_TEMPLATE/1-feature-request.yml

```diff
@@ -1,6 +1,6 @@
-name: '📝 Custom issue'
-description: 'Custom issue template for plugin development'
-labels: 'enhancement'
+name: '📝 Feature Request'
+description: 'feature request template for plugin development'
+labels: ['enhancement']
```

Comment:
- According to the new [labels](https://github.com/jenkinsci/ecu-test-execution-plugin/blob/d37b6049def956a614c7f15d251d2141330a73cc/.github/labels.yml#L25)  schema, there is no enhancement anymore
- Descriptions should start with a capital